model/role/db.go

```diff
@@ -0,0 +1,39 @@
+package role
```

Comment:
why not put this in the model-user collection?

Reply:
These documents have a completely different structure. I moved it into the user package though